insta_reels_publishing_api_sample/index.js

```diff
@@ -22,9 +26,24 @@ const {
     PORT,
     REDIRECT_URI,
     APP_ID,
-    API_SECRET
+    API_SECRET,
+    GRAPH_API_ORIGIN,
+    GRAPH_API_VERSION_PATH,
 } = process.env;
 
+const GRAPH_API_BASE_URL = (GRAPH_API_ORIGIN ?? DEFAULT_GRAPH_API_ORIGIN) +
+    (GRAPH_API_VERSION_PATH ?? DEFAULT_GRAPH_API_VERSION_PATH);
```

Comment:
Shouldn't there be a `/` between `GRAPH_API_ORIGIN` and `GRAPH_API_VERSION_PATH`?

Reply:
For example, it should be `https://graph.facebook.com/v18.0/`, not `https://graph.facebook.comv18`

Reply:
Tweaked to ensure the `/` are included.